flexmeasures/cli/tests/test_utils.py

```diff
@@ -69,3 +69,45 @@ def test_get_timerange_from_flag(monkeypatch, now, flag, expected_start, expecte
 
     assert start == expected_start
     assert end == expected_end
+
+
+def test_get_unique_sensor_names(app, db, add_asset_with_children):
+    from flexmeasures.cli.utils import get_sensor_aliases
+    from flexmeasures.cli.data_show import find_duplicates
```

Comment:
Move to top?

Reply:
I couldn't move them because, otherwise, the `RuntimeError: Working outside of application context.` error showed up.